Add design documents (#1)
cybozu-go/myso: docs/crd_mysql_user.md

User
====

> ** NOTE "": This custom resource is currently not used.

Reviewer: ```suggestion
**NOTE**: This custom resource will not be implemented soon.
```
